test/kernels/keops/test_matern_kernel.py

```diff
@@ -19,34 +21,56 @@ def create_kernel_ard(self, num_dims, **kwargs):
             return MaternKernel(nu=2.5, ard_num_dims=num_dims, **kwargs)
 
     class TestMaternKeOpsKernel(unittest.TestCase):
+
+        # keops tests
```

Comment:
It would be nice to put all of these non-BaeKernelTestCase tests into some super class. It looks like the same tests are essentially repeated for Matern, Periodic, and RBF kernels, and unifying these tests in a superclass will lead to easier maintenance.

Reply:
I've now added a `BaseKeOpsTestCase` class that has all the common tests.